README.md

```diff
@@ -1,62 +1,77 @@
 
+[https://docs.google.com/document/d/1N8cYiQC9g8rheZzI37Eo2iKghC6C3oKZ069Ttv6H61o/edit](https://docs.google.com/document/d/1N8cYiQC9g8rheZzI37Eo2iKghC6C3oKZ069Ttv6H61o/edit)
 
 ## How to contribute
 
 0. Join Discord at https://discord.gg/yvtBmEW and ask any questions in the #cb-connect-questions channel.
 1. Fork this repository and download your copy with `git clone [YOUR_PATH]`. You can also request to be added as a collaborator.
-2. Install meteor: https://www.meteor.com/install
-3. Before you start up the app, make sure to `meteor npm install`.
-4. Start the app with the following command: `meteor --settings dev-settings.json`
+1. Install meteor: https://www.meteor.com/install
+1. Before you start up the app, make sure to `meteor npm install`.
+1. Start the app with the following command: `meteor --settings dev-settings.json`
 
 Tips:
-- If you need help or would like to pair on an issue, join our Discord at https://discord.gg/yvtBmEW ask! 
+
+- If you need help or would like to pair on an issue, join our Discord at https://discord.gg/yvtBmEW ask!
 - When you have localhost:3000 running, type control-m to see the local data available on the page.
 
 Feel free to file new issues. We're looking for design, UX, and code (any: React, Javascript, Meteor, MongoDB) help.
 
 If you see an issue under the `help wanted` category that you'd like to tackle, please comment that you're working on it, and create a new branch for the issue.
 
-Once the prototype is launched, we'll also need volunteer moderators/matchers. 
+Once the prototype is launched, we'll also need volunteer moderators/matchers.
 
 Please submit new pull requests against the `staging` branch.
 
 ### How to login on localhost
-After you create a new user by submitting an entry, you'll be able to log in by typing in the email you used to apply, and "password" for the password. 
 
-If you want to get the verification email, please create an API key in Sparkpost and put it into line 14 in dev-settings.json. 
+After you create a new user by submitting an entry, you'll be able to log in by typing in the email you used to apply, and "password" for the password.
+
+If you want to get the verification email, please create an API key in Sparkpost and put it into line 14 in dev-settings.json.
+
+#### How to deploy to production
+
+- ssh into DO server
+- cd into `repos/cb-connect`
+- pull the latest changes `git pull`
+- cd into `repos/cb-connect/scripts/` and run `./build.sh`
+  this will create a bundle file inside `/cb-connect/bundles`
+- When `./build.sh` script will run successfully, it will output the deploy command like: `./deploy.sh XYZ` where XYZ is the commit hash.
+- Copy and run the deploy command, it will deploy the bundle. In case of emergencies or failed deployment you can rollback the last release by running `./deploy.sh` script with previous hash.
```

Comment:
Make sure to remove the bundle files inside /cb-connect/bundles after it grows to ~100 or more.

Reply:
TODO: Move mongo creds from ./deploy.sh to settings.json.